tck/README.adoc

```diff
@@ -53,13 +53,13 @@ To enable the tests in your project you need to add the following dependency to
 <dependency>
     <groupId>org.testng</groupId>
     <artifactId>testng</artifactId>
-    <version>7.4.0</version>
+    <version>7.8.0</version>
```

Comment:
These dependency are from tck parent. You don't need to specify the version.

Reply:
Ah, perfect, I'll remove the explicit version.